tests/integration/test_load_and_run_multi_agent.py

```diff
@@ -23,29 +28,33 @@ def test_load_and_run_multi_agent(
             f"Skipping test for {agent_framework.name} because it does not support multi-agent"
         )
 
-    kwargs["model_id"] = "gpt-4.1-nano"
+    agent_model = "gpt-4.1-nano"
+    # kwargs["model_id"] = "gemini/gemini-2.0-flash-lite"
+    # kwargs["model_id"] = "gemini/gemini-2.5-flash-preview-04-17"
+    kwargs["model_id"] = agent_model
```

Comment:
This will be reverted prior to merge right? Same question for the commented out model_args a few lines below